File: code/java/src/main/java/adapter/Traveler.java
package adapter;

class Traveler {
    public static void main(String[] args) {
        // Create electronic device
        MobilePhone device = new MobilePhone();

        // Create universal adapter
        UniversalAdapter adapter = new UniversalAdapter(device);

        // Connect device to electrical outlet and charge
        adapter.request();
    }
}

File: code/python/adapter/Traveler.py
from adapter.MobilePhone import MobilePhone
from adapter.UniversalAdapter import UniversalAdapter

# To run the code
if __name__ == "__main__":
    # Create electronic device
    device = MobilePhone()

    # Create universal adapter
    adapter = UniversalAdapter(device)

    # Connect device to electrical outlet and charge
    adapter.request()
